assembl/tests/views_tests/test_graphql_vote_session.py

```diff
@@ -0,0 +1,254 @@
+# -*- coding: utf-8 -*-
+
+from assembl.graphql.schema import Schema as schema
+
+
+def snake_to_camel(string):
+    # https://stackoverflow.com/questions/19053707/converting-snake-case-to-lower-camel-case-lowercamelcase
+    components = string.split('_')
+    # We capitalize the first letter of each component except the first one
+    # with the 'title' method and join them together.
+    return components[0] + "".join(x.title() for x in components[1:])
+
+
+def assert_langstring_is_equal(langstring_name, fetched_model, source_model):
+    source_value = getattr(source_model, langstring_name).entries[0].value
```

Comment:
Matching the `0`th value between `langstring` entries is extremely unsafe! You should make a check on the the specific locale, OR, check the `first_original` of the langstrings to check both versions.

Reply:
I understand in testing this might not be a real issue actually, so perhaps make a comment about this.